test/runners/test_aws_batch.js

```diff
@@ -10,11 +10,14 @@ describe('AWS Batch createRunner tests', () => {
     beforeEach(() => {
         this.getStub = sinon.stub(axios, 'get').returns(Promise.resolve({ data: JSON.stringify({ region: 'test-reg' }) }));
         this.addExceptionStub = sinon.stub(tracer, 'addException');
-        config.setConfig({ metadataOnly: false });
+        this.getConfigStub = sinon.stub(config, 'getConfig').returns({
+            metadataOnly: false, // adding only members relevant to the test.
```

Comment:
This is not so good... you assume that this the only field in the metadata that is used

Reply:
the tests are creating a stub with only the relevant data to the test. If more data will be relevant - we will add it